lib/matplotlib/tests/test_colorbar.py

```diff
@@ -979,6 +979,37 @@ def test_colorbar_extend_drawedges():
             np.testing.assert_array_equal(cbar.dividers.get_segments(), res)
 
 
+@image_comparison(['contourf_extend_patches.png'], remove_text=True,
+                  style='mpl20')
+def test_colorbar_contourf_extend_patches():
+    params = [
+        ('both', 5, ['\\', '//']),
+        ('min', 7, ['+']),
```

Comment:
This may seem stupid, but this did actually break at one stage as hatches wasn't extended enough.